package.json

```diff
@@ -8,7 +8,7 @@
   "version": "0.52.1",
   "files": [
     "dist",
-    "lib/css/exports/*.less"
+    "lib/css"
```

Comment:
Got it. I want those Less files to be available to Core and Careers if we deliver these things via `npm` there. :v:

Reply:
Sounds good to me. We might want to rethink how we export singular mixins, etc. I remember once wanting to use the button mixing and changing some colors around for some custom marketing pages, but was completely unable to do so because of some internal weirdness in our dependency chain.

Not a problem for right now, but closer to a 1.0 launch.